plugins/modules/keycloak_userprofile.py

```diff
@@ -0,0 +1,608 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) Ansible project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: keycloak_userprofile
+
+short_description: Allows managing Keycloak User Profiles.
```

Comment:
This is the only description which shouldn't end with a period :)
```suggestion
short_description: Allows managing Keycloak User Profiles
```

Reply:
:+1: